src/pages/admin/milestone.tsx

```diff
@@ -0,0 +1,74 @@
+import { SignInButton, SignInFormItem, SignInTitle } from '@/components/auth';
+import { VerticalForm } from '@/components/common';
+import { config } from '@/lib';
+import withAccessType from '@/lib/hoc/withAccessType';
+import { PermissionService } from '@/lib/services';
+import type { GetServerSideProps, NextPage } from 'next';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { AiOutlineMail } from 'react-icons/ai';
+import { VscLock } from 'react-icons/vsc';
+
+interface FormValues {
+  name: string;
+  points: number;
+}
+const AwardPointsPage: NextPage = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormValues>();
+
+  const onSubmit: SubmitHandler<FormValues> = () => {
+    // TODO
+  };
+
+  return (
+    <VerticalForm onEnterPress={handleSubmit(onSubmit)}>
+      <SignInTitle
+        text="Create Milestone"
+        description="Creating a Milestone Will Award Points to All User Active Accounts"
+      />
+      <SignInFormItem
+        icon={<AiOutlineMail />}
```

Comment:
I see that you've copied this from the sign-in page, which is fine, but it might be confusing even for the people using the admin page if it looks too much like the sign-in page. could at least remove the mail/password icons